### PR TITLE
done

### PR DESCRIPTION
**Pull request by Nazar-Brunarskyi**
[DEMO LINK](https://Nazar-Brunarskyi.github.io/react_todo-app-with-api/)

### CODE REVIEW

**Comment by Nazar-Brunarskyi** [comment]
*

File: src/components/NewTodo.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      onTitleSubmit,
+    } = useContext(GlobalContext);
+
+    const {
+      isToggleAllActive,
+      onToggleAll,
+    } = useContext(EditContext);
+
+    const TodoField = useRef<HTMLInputElement>(null);
+    const [inputValue, setInputValue] = useState('');
```
Comment (Ihor-Karpyn) [inline src/components/NewTodo.tsx:27]: var name

File: src/contexts/GlobalContext.tsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  setTemporaryTodo: React.Dispatch<React.SetStateAction<Todo | null>>,
+  showError: (errorTextToShow: string) => void,
+  onTitleSubmit: (title: string) => void,
+  visibleTodos: Todo[],
+}
+
+export const GlobalContext = createContext<ContextType>({
+  todos: [],
+  setTodos: () => { },
+  noError: true,
```
Comment (Ihor-Karpyn) [inline src/contexts/GlobalContext.tsx:37]: boolean var should start from `is` or should

**Review by Ihor-Karpyn — APPROVED** [review]
GJ

**Pull request closed without merging**